.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 node_js:
-  - 4
```

Comment:
We depend on file-disk which uses default parameters in some functions. This doesn't work in node4: http://node.green/#ES2015-syntax-default-function-parameters